features/util.py

```diff
@@ -113,13 +114,24 @@ def repo_state_hash(include_features: bool = False):
         git repository
     """
     git_rev_cmd = ["git", "rev-parse", "HEAD"]
+    rev_out = subprocess.check_output(git_rev_cmd)
+
     git_diff_cmd = ["git", "diff"]
     if not include_features:
         files_and_folders = [f for f in os.listdir(".") if "features" not in f]
         git_diff_cmd += ["--", *files_and_folders]
-    rev_out = subprocess.check_output(git_rev_cmd)
     diff_out = subprocess.check_output(git_diff_cmd)
-    output_to_hash = rev_out + diff_out
+
+    git_status_cmd = ["git", "status", "--porcelain"]
```

Comment:
I see why we are doing this, but I don't know if we should. Any test script that we create or any manual test we add, will trigger for a new package build. I know this is the more complete solution, but I don't know if it pays off.

What do you think @renanrodrigo @blackboxsw ?

Reply:
I updated this function to exclude more directories

Reply:
I think it is fine. We can tweak the "watched" directories in the future if needed, but so far, it works